networkit/test/test_centrality.py

```diff
@@ -4,11 +4,92 @@
 import networkit as nk
 import numpy as np
 import scipy
-
+import random
 
 class Test_Centrality(unittest.TestCase):
 
-	def test_DegreeCentrality(self):
+	def checkCovers(self, c1, c2):
+		if not c1.numberOfElements() == c2.numberOfElements(): return False
+		if not c1.numberOfSubsets() == c2. numberOfSubsets(): return False
+		for i in range(0,c1.numberOfElements()):
+			if not c1.subsetsOf(i) == c2.subsetsOf(i): return False
+		return True
+	
+	def setUp(self):
+		self.L = nk.readGraph("input/looptest1.gml", nk.Format.GML) #without self-loops
+		self.LL = nk.readGraph("input/looptest2.gml", nk.Format.GML) #with self-loops sprinkled in
+	
+	def testCentralityApproxBetweenness(self):
```

Comment:
I think we can delete the module name from the function definition without loosing information. On the command line a single test execution looks like this: 

```
testCentralityCoreDecomposition (test_centrality.Test_Centrality.testCentralityCoreDecomposition) ... ok
```